roman.py

```diff
@@ -1,13 +1,17 @@
+import re
+
 def roman_to_integer(value):
     number_dict = {'I':1, "V":5, "X":10, "L":50}
```

Comment:
For security could be good change this structure to be a tuple instead of dict. Dicts are mutable and tuple isn't.
Use UPPER Case and put out of the function definition the variable instance.
Something like:
`ROMANS_NUMBER = (('I', 1), ('V', 5), ...)
def roman_to_integer(value):
   ...
`

Reply:
This is a great idea, thanks.